frontend/src/components/ApplicationForm/index.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Container, CardMedia, Typography } from "@mui/material";
+import Table from "@mui/material/Table";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import { isLoggedIn } from "../../utils";
+import {
+  UserInfoTypography,
+  Section,
+  SectionHeader,
+  CampaignImageCard,
+  RoleButton,
+  Question,
+  Answer,
+  CampaignDescription,
+  FormContainer,
+  AuthLink,
+  UserInfoCell,
+  SpaceRight,
+} from "./applicationForm.styled";
+
+const ApplicationForm = (props) => {
+  const {
+    questions,
+    roles,
+    rolesSelected,
+    setRolesSelected,
+    answers,
+    setAnswers,
+    campaignName,
+    headerImage,
+    description,
+    userInfo,
+  } = props;
+
+  const loggedIn = isLoggedIn();
+
+  // FIXME: part of CHAOS-51, handle single question related to many roles
+  //        (currently gets printed under each rolename and all are overwritten)
+  const handleAnswerInput = (e, qID) => {
+    setAnswers({ ...answers, ...{ [qID]: e.target.value } });
+  };
+
+  const toggleRole = (id) => {
+    if (rolesSelected.includes(id)) {
+      setRolesSelected(rolesSelected.filter((role) => role !== id));
+    } else {
+      setRolesSelected((existing) => [...existing, id]);
+    }
+  };
+  return (
+    <Container>
+      <CampaignImageCard sx={{ boxShadow: 15 }}>
```

Comment:
better to set the default style for `CampaignImageCard` to 15 (from 20) rather than inline it here